code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -49,39 +49,31 @@
 		if(15)
 			new /obj/item/nullrod/armblade(src)
 		if(16)
-			new /obj/item/guardiancreator/random(src)
-		if(17)
 			if(prob(50))
 				new /obj/item/disk/design_disk/modkit_disc/mob_and_turf_aoe(src)
 			else
 				new /obj/item/disk/design_disk/modkit_disc/bounty(src)
-		if(18)
+		if(17)
 			new /obj/item/warp_cube/red(src)
+		if(18)
+			new /obj/item/organ/heart/gland/heals
 		if(19)
-			new /obj/item/wisp_lantern(src)
```

Comment:
Undocumented change also why remove the wisp lantern?

Reply:
‘This change is undocumented, saying that you removed it doesn’t count’

Do you have brain damage

Reply:
Just add it to the changelog is what I meant sorry